### PR TITLE
LPS-119812 [Content Performance Panel] Delete unneeded code form AnalyticsReportsDisplayContext

### PR DESCRIPTION
**Pull request by cgoncas**
**Description of the pull**

We have moved all the needed information for the metrics react component to the get_data endpoint, so now we can delete the old code form the AnalyticsReportDisplayContext.

### CODE REVIEW

**Comment by cgoncas** [comment]
ci:test:sf

**Comment by cgoncas** [comment]
ci:test:relevant

**Comment by cgoncas** [comment]
Hey @dgarciasarai  and @darquesdev !!

Could you test this pull? sometimes I receive a strange error. But I am starting to think this error is related to my installation.

Thanks!!!

File: modules/dxp/apps/analytics/analytics-reports-web/src/main/java/com/liferay/analytics/reports/web/internal/portlet/action/GetDataMVCResourceCommand.java [hunk 1]
```diff
@@ -179,32 +181,31 @@ private JSONObject _getAuthorJSONObject(
...
+		if (portraitId > 0) {
+			authorProfileImage = webImage.getUrl();
 		}
-		catch (PortalException portalException) {
-			_log.error(portalException, portalException);
 
-			return JSONFactoryUtil.createJSONObject();
-		}
+		return JSONUtil.put(
+			"alt", webImage.getAlt()
```
Comment (dgarciasarai) [inline modules/dxp/apps/analytics/analytics-reports-web/src/main/java/com/liferay/analytics/reports/web/internal/portlet/action/GetDataMVCResourceCommand.java:203]: What do you think about rename this to `name`? (also in frontend)
```java
"name", webImage.getName()
```
From @carloslancha feedback.

File: modules/dxp/apps/analytics/analytics-reports-web/src/main/java/com/liferay/analytics/reports/web/internal/display/context/AnalyticsReportsDisplayContext.java [hunk 2]
```diff
@@ -131,129 +85,12 @@ public String getLiferayAnalyticsURL() {
...
-					DateTimeFormatter.ISO_DATE.format(
-						defaultTimeRange.getStartLocalDate())
-				).build();
-			}
-		).put(
-			"defaultTimeSpanKey", TimeSpan.defaultTimeSpanKey()
-		).put(
 			"endpoints",
 			HashMapBuilder.<String, Object>put(
 				"getAnalyticsReportsData",
```
Comment (dgarciasarai) [inline modules/dxp/apps/analytics/analytics-reports-web/src/main/java/com/liferay/analytics/reports/web/internal/display/context/AnalyticsReportsDisplayContext.java:90]: From @carloslancha feedback:
>To me anything starting with get is a method (a getter), getAnalayticsReportsDataURL sounds like something that will return you an URL. Something like analyticsReportsDataURL. analyticsReportsDataFetchURL, analyticsReportsDataEndpointURL, makes more sense to me in this case.

Maybe we should rename this (also in the frontend side) to something like he proposed 🤔  `analyticsReportsDataURL`?

**Review by dgarciasarai — COMMENTED** [review]
It works! Thanks @cgoncas! Remove code it's a pleasure 😂 

I added only two comments about naming, nothing blocking. 😄 

Screenshot 2020-08-26 at 15 55 20
Screenshot 2020-08-30 at 16 05 46

File: modules/dxp/apps/analytics/analytics-reports-web/src/main/java/com/liferay/analytics/reports/web/internal/display/context/AnalyticsReportsDisplayContext.java (hunk 2 above)
Comment (cgoncas) [reply modules/dxp/apps/analytics/analytics-reports-web/src/main/java/com/liferay/analytics/reports/web/internal/display/context/AnalyticsReportsDisplayContext.java:90]: I totally agree, could you open an impeding with this?

File: modules/dxp/apps/analytics/analytics-reports-web/src/main/java/com/liferay/analytics/reports/web/internal/portlet/action/GetDataMVCResourceCommand.java (hunk 1 above)
Comment (cgoncas) [reply modules/dxp/apps/analytics/analytics-reports-web/src/main/java/com/liferay/analytics/reports/web/internal/portlet/action/GetDataMVCResourceCommand.java:203]: Awesome!!, could you open an impeding bug with this?

**Comment by cgoncas** [comment]
ci:test:sf

**Comment by cgoncas** [comment]
cit:test:relevant

File: modules/dxp/apps/analytics/analytics-reports-web/src/main/java/com/liferay/analytics/reports/web/internal/display/context/AnalyticsReportsDisplayContext.java (hunk 2 above)
Comment (dgarciasarai) [reply modules/dxp/apps/analytics/analytics-reports-web/src/main/java/com/liferay/analytics/reports/web/internal/display/context/AnalyticsReportsDisplayContext.java:90]: Yes! Do you want one impedibug for both changes or split into two?

File: modules/dxp/apps/analytics/analytics-reports-web/src/main/java/com/liferay/analytics/reports/web/internal/portlet/action/GetDataMVCResourceCommand.java (hunk 1 above)
Comment (dgarciasarai) [reply modules/dxp/apps/analytics/analytics-reports-web/src/main/java/com/liferay/analytics/reports/web/internal/portlet/action/GetDataMVCResourceCommand.java:203]: Yes! Can you answer me this? https://github.com/liferay-tango/liferay-portal/pull/203#discussion_r477409702 😄

File: modules/dxp/apps/analytics/analytics-reports-web/src/main/java/com/liferay/analytics/reports/web/internal/display/context/AnalyticsReportsDisplayContext.java (hunk 2 above)
Comment (cgoncas) [reply modules/dxp/apps/analytics/analytics-reports-web/src/main/java/com/liferay/analytics/reports/web/internal/display/context/AnalyticsReportsDisplayContext.java:90]: One impedibug looks ok for me :)

Comment (dgarciasarai) [reply modules/dxp/apps/analytics/analytics-reports-web/src/main/java/com/liferay/analytics/reports/web/internal/display/context/AnalyticsReportsDisplayContext.java:90]: https://issues.liferay.com/browse/LPS-119831

**Comment by dgarciasarai** [comment]
ci:test:relevant

**Comment by cgoncas** [comment]
ci:forward

**Pull request closed without merging**